### PR TITLE
chore(docker): rename `npx playwright docker` to `npx playwright container`

### PR DESCRIPTION
**Pull request by aslushnikov**
Drive-by: fix smoke container tests

### CODE REVIEW

File: .github/workflows/tests_primary.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ jobs:
     - run: npx playwright install --with-deps
     - run: |
         ./utils/docker/build.sh --amd64 focal $PWTEST_DOCKER_BASE_IMAGE
-        npx playwright docker build
+        npx playwright ctr build
```
Comment (mxschmitt) [inline .github/workflows/tests_primary.yml:149]: this is not a common abbreviation for container, never seen this before, I'd keep it `container`.

Comment (aslushnikov) [reply .github/workflows/tests_primary.yml:149]: sure, `container` it is!

Comment (mxschmitt) [inline .github/workflows/tests_primary.yml:149]:
```suggestion
        npx playwright container build
```

File: .github/workflows/tests_secondary.yml [hunk 2]
```diff
@@ -817,7 +817,7 @@ jobs:
     - run: npm run build
     - run: npx playwright install --with-deps
     - run: ./utils/docker/build.sh --amd64 focal playwright:localbuild
-    - run: PWTEST_DOCKER_BASE_IMAGE=playwright:localbuild npx playwright docker build
+    - run: PWTEST_DOCKER_BASE_IMAGE=playwright:localbuild npx playwright ctr build
```
Comment (mxschmitt) [inline .github/workflows/tests_secondary.yml:820]:
```suggestion
    - run: PWTEST_DOCKER_BASE_IMAGE=playwright:localbuild npx playwright container build
```

**Pull request closed without merging**